velocity-transition-group.js

```diff
@@ -44,6 +44,13 @@ var ReactDOM = require('react-dom');
 var ReactTransitionGroup = require('react-addons-transition-group');
 var Velocity = require('./lib/velocity-animate-shim');
 
+// Shim requestAnimationFrame for browsers that don't support it, in particular IE 9.
+var shimRequestAnimationFrame = 
```

Comment:
Sorry, I didn't think of the server case. So shimRequestAnimationFrame can just be set to null if window is undefined?